File: problemas/iniciante/2061/2061.exs
defmodule Modulo do
  def processa(n, m) do
    if m > 0 do
      acao = IO.gets("") |> String.trim
      if acao == "fechou" do
        processa(n + 1, m - 1)
      else
        processa(n - 1, m - 1)
      end
    else
      n
    end
  end
end

entrada = IO.gets("") |> String.trim |> String.split(" ")
n = entrada |> Enum.at(0) |> String.to_integer
m = entrada |> Enum.at(1) |> String.to_integer

IO.puts Modulo.processa(n, m)

File: problemas/iniciante/2061/2061.rb
entrada = gets.chomp.split.map(&:to_i)
n = entrada[0]
m = entrada[1]

for i in 1..m do
  acao = gets.chomp
  if acao == "fechou" then
    n += 1
  else
    n -= 1
  end
end

puts n